aws-go-eks/README.md

```diff
@@ -8,9 +8,10 @@ This example creates an AWS EKS Cluster and deploys a sample container applicati
 ### Prerequisites
 
 1. [Install Pulumi](https://www.pulumi.com/docs/get-started/install/)
-2. [Install Node.js 8.11.3](https://nodejs.org/en/download/)
+2. [Install Go 1.13](https://golang.org/doc/install)
```

Comment:
Why is go 1.13 needed for csharp? Is this a dep?

Reply:
that's the Go README ;)

Reply:
Ah, makes more sense now 👍